app/helpers/application_helper.rb

```diff
@@ -37,6 +37,7 @@ def render_categories(categories)
         )
       end
       tree_element << category.name
+      tree_element << content_tag("span", category.tags.map(&:name).map(&:humanize).map(&:titleize).join(", "), class: "badge badge-light")
```

Comment:
the names in role section are not generated by a defined list or defined in "i18n". Aligning that or setting it up cleanly in both places certainly makes sense, but it might go beyond the scope here?!

I will see if i can find a simply solution.

Reply:
Now using I18N for DataResource names

<img width="563" alt="Bildschirmfoto 2024-05-02 um 14 43 26" src="https://github.com/smart-village-solutions/smart-village-app-mainserver/assets/90779/3282204c-f4bd-4ffd-8ee8-b212506cea4b">